### DISCUSSION TITLE
Is it possible to extend the category / collection object?

### DISCUSSION THREAD

**Discussion by peteror** [saleor/saleor#12725] [Ideas]
I'm running a store where the main (only) access point for our customers is a mobile app.  I'm looking for ways to include more information with categories / collections, on top of the single image and text field. (for example, a category could have a different image for a banner, an icon and an actual background image on the category listing page) 

Is it possible to extend categories / collections with attributes, the same way I can do it with products? 
Alternatively, could we have the same field types for metadata than we have with attributes? (adding image URLs to metadata is doable, but not very user-friendly).

**Comment by timuric** [comment]
Hi @peteror  yes we have plans to add attributes to all entities, but I do not have estimates when we will ship it, perhaps in Q3 2023

**Reply by madmonkeyworks** [reply]
Hi there, any idea when this feature will be available? Having attributes on categorories and collections would be great.

**Reply by timuric** [reply]
We plan to add it in Q1 2024, meantime you can use metadata
